### ISSUE TITLE
NOTICE: This project temporarily uses a forked version of joern

### ISSUE THREAD

**Issue by prabhu** [AppThreat/cpggen#31]
After detailed discussion with joern developers, I am using scala 3 branch of joern to bring some new features and bug fixes

https://github.com/AppThreat/joern2

After some exhaustive testing, I'm confident this approach is working well for the use cases and projects I am aware of. I will raise another ticket once the project is ready to switch back to the upstream version which might be in a few weeks time.
